package.json

```diff
@@ -27,6 +27,8 @@
     "release": "yarn build && changeset publish"
   },
   "resolutions": {
+    "**/@typescript-eslint/eslint-plugin": "^4.22.0",
+    "**/@typescript-eslint/parser": "^4.22.0",
```

Comment:
any particular reason for those resolutions here? 🤔

Reply:
`gatsby` has a dependency on `^2.x` for each, and it was causing linting to fail. I can investigate more to see if there's a better way, this was just the quick and easy solution I Google'd to avoid the conflict.

Reply:
Hm, why does Gatsby depend on this anyway? It shouldn't be concerned with linting by default 🤔

Reply:
I removed the `resolutions` and `node_modules` and reinstalled and it seems to be working fine now. I'm not sure what happened there exactly, though `gatsby` fixed that issue in February so maybe I had an outdated version installed.